src/backend/commands/player/misc/tp_to_player.cpp

```diff
@@ -0,0 +1,65 @@
+#include "backend/player_command.hpp"
+#include "util/teleport.hpp"
+
+namespace big
+{
+	class tp_to_player : player_command
+	{
+		using player_command::player_command;
+
+		virtual std::optional<std::vector<std::string>> get_argument_suggestions(int arg) override
+		{
+			if (arg == 1 || arg == 2)
+			{
+				std::vector<std::string> suggestions;
+				for (auto& [_, player] : g_player_service->players())
```

Comment:
If you're not making use of the key then consider reworking the for loop to make use of `std::ranges::views::values`.

```cpp
for (const auto& session_type_string : m_session_types | std::ranges::views::values)
{

}
```

Reply:
Unused variable `_`.